gear/gear/clients.py

```diff
@@ -1,6 +1,6 @@
 from typing import Optional
```

Comment:
This file also feels like the sort of thing that should be an interface with an instance per cloud and you allocate it once at start up time.